docs/pages/upgrading/cloud-linux.mdx

```diff
@@ -164,16 +164,115 @@ automatic updates:
    $ sudo systemctl status teleport-upgrade
    ```
 
+## Version locking
+
+Due to the fact that Teleport has strict [version compatibility requirements](../faq.mdx#which-version-of-teleport-is-supported),
+the updater requires full control over the version of Teleport. As of Teleport
```

Comment:
> Due to the fact that Teleport has strict [version compatibility requirements](../faq.mdx#which-version-of-teleport-is-supported),
the updater requires full control over the version of Teleport.

Reading the compatibility requirements is already a prerequisite of the guide, so I think we can remove this sentence.